plugin/src/main/java/at/helpch/chatchat/event/ChatChatEvent.java

```diff
@@ -0,0 +1,71 @@
+package at.helpch.chatchat.event;
+
+import at.helpch.chatchat.format.ChatFormat;
+import net.kyori.adventure.audience.Audience;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class ChatChatEvent extends Event implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
```

Comment:
handlerList should be static, along with  a static getHandlerList() method as per spigot wiki https://www.spigotmc.org/wiki/using-the-event-api/